### ISSUE TITLE
Add P2A as an output type

### ISSUE THREAD

**Issue by glozow** [mempool/mempool#5492]
Filing ahead of v28.0 release as a notification of a standardness rule change. (The demo transactions aren't meant to show a "bug" since the new rule isn't in any released version, they are to provide test cases for somebody implementing this feature request.)

[Bitcoin Core #30352](https://github.com/bitcoin/bitcoin/pull/30352) added P2A as an output type, meaning a spk with `OP_1 <4e73>` is now `ANCHOR` instead of `UNKNOWN`.
I haven't looked at the code to see how the types are determined, but using Bitcoin Core, `bitcoin-cli decodescript 51024e73` should give you a JSON result with `"type": "anchor"` instead of `"type": "witness_unknown"`

As a test case, see Type for p2a address on testnet4: https://mempool.space/testnet4/address/tb1pfees9rn5nz

![image](https://github.com/user-attachments/assets/2824a3c4-ebad-46c5-addb-f213dd0022b0)

And a test case for mainnet: https://mempool.space/address/bc1pfeessrawgf

![image](https://github.com/user-attachments/assets/00345826-a110-4894-a783-bc54ea99d7e0)

Additionally, spending these outputs in a transaction will no longer be nonstandard.

As a test case, see https://mempool.space/testnet4/tx/4a89d5d1568b5cbcb4118559cb65d2357657d361a41cd96b14e74ed3d065975c which is currently labeled nonstandard.

![image](https://github.com/user-attachments/assets/644d0295-045a-4378-b975-c7101d7bc768)

Preferably, when it becomes standard, the node should accept these transactions so they can be displayed before confirmation. Upgrading the node to v28.0 (when it is released) will resolve this.

In summary, there are 3 requests here:
- [ ] Add `ANCHOR` as an address/output type for spk 50124e73
- [ ] Don't display anchor spends as non-standard
- [ ] Upgrade node to v28.0 (when available) so that users can see unconfirmed transactions spending p2a outputs

**Comment by russeree** [comment]
Possible further anchor visualization with Goggles.

https://github.com/mempool/mempool/pull/5768
